client/components/chart/index.jsx

```diff
@@ -60,29 +61,27 @@ module.exports = React.createClass( {
 	},
 
 	resize: function() {
-		if ( this.isMounted() ) {
```

Comment:
I'm more curious as to why we needed the check in the first place. 👍 Let's remove this for now and see if we get any related logged error events when this PR is merged